### ISSUE TITLE
Is there a way to add PHP template name support for custom render function?

### ISSUE THREAD

**Issue by Anubarak** [Haehnchen/idea-php-symfony2-plugin#1704]
Sorry for the incredible bad title.. and thank you very much for your great plugin, it really helps me.
One thing I was looking for is the following:

My Controller
```PHP
// works...
public function myFunction()
{
    $this->view->render('template-path.twig, ['var1' => 1]);
}
```
pressing CTRL enables me to jump right to the template and all variables in the template are available in PHPStorm
![image](https://user-images.githubusercontent.com/6122122/133041080-239bfda3-990e-4083-88c4-9ebdce6d82cf.png)
![image](https://user-images.githubusercontent.com/6122122/133041271-903bd4d2-20f1-41ce-bb1c-17f7195a9c29.png)

I would like to create a custom wrapper for this

```PHP
public function myCustomTemplateRenderFunction(string $templateName, array $variables = [])
{
    // do some stuff, proceed the templateName and variables... custom logic here...

    return $this->view->render('calculations/' . $templateName . '.twig', $variables);
}

// .....
// automatically jumping to the template or variable suggestions is not possible here anymore
$this->myCustomTemplateRenderFunction('test', ['var1' => 1]);
```
So my question is: is there any possible way (via PHPDoc, Plugin Settings injection or whatever) to tell PHPStorm `myCustomTemplateRenderFunction` actually calls a render function with a certain path in the end?

Thank you very much.

**Comment by Lorenzschaef** [comment]
Maybe this is what you are looking for?

https://www.jetbrains.com/help/phpstorm/symfony-creating-helper-functions.html

**Comment by Anubarak** [comment]
@Lorenzschaef Not 100% what I wanted - but I can go with that. Thank you very much. This functionality is awesome

**Comment by Haehnchen** [comment]
yes, in general i am thinking to support "wildcards" for templates in many places and also to improve all php completion with code path analyse, with both this would catch your use case here.
